### PR TITLE
Cc/connect tools

### PR DESCRIPTION
**Pull request by celaus**

### CODE REVIEW

File: pages/docs/clients/index.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
-title: Crate Client Libraries
+title: Client Libraries & Applications
 author: Chris Ward
 
-Crate can be part of any application stack, with official and community maintained client libraries that support a multitude of programming languages.
+Crate is happy at the heart of any application stack, integrating with a multitude of languages and applications to connect and query your data.
 
-## Officially Maintained Clients
+## Officially Maintained Client Libraries
```
Comment (chaudum) [inline pages/docs/clients/index.html]: suggestion: "Official Client Libraries"

File: pages/docs/tools/index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
+title: Tools Overview
+description: Management tools, utilities and data visualization for Crate
+author: Christian Haudum
+
+
+There are several tools and utilities that work with Crate.
```
Comment (chaudum) [inline pages/docs/tools/index.html]: I think this first "paragraph" can be improved, something like:

there are various different tools and utilities that can connect to crate, ranging from ... to .... 
they can do this and that, ... use JDBC standard .... HTTP

I think you know what I mean :)

File: templates/docs.html [hunk 3]
```diff
@@ -65,7 +65,7 @@
 	</ul> -->
 	{% endif %}
 </li>
-<li class="navleft-item"><a href="/docs/clients/">Client Libraries</a>
+<li class="navleft-item"><a href="/docs/clients/">Client Libraries & Applications</a>
```
Comment (chaudum) [inline templates/docs.html]: please use `&amp;`

File: templates/docs.html [hunk 4]
```diff
@@ -77,6 +77,7 @@
 	</ul>
 	{% endif %}
 </li>
+<li class="navleft-item"><a href="/docs/tools/">Tools</a></li>
```
Comment (chaudum) [inline templates/docs.html]: please use `url` template tag when referencing pages within cactus site.
`{% url '....html' %}`
this will warn you if the referenced page is not available
you can do a separate PR to change that in this template file, though.

**Comment by chaudum** [comment]
move, not remove: `pages/docs/clients/squirrel.html` -> ` pages/docs/tools/squirrel.html`

File: templates/docs.html (hunk 4 above)
Comment (celaus) [inline templates/docs.html]: I am not sure if this still applies since all other URLs are not like this and we manage quite a lot of the things with absolute URLs and redirects ....

**Comment by celaus** [comment]
@ squirrel page: why? nothing links to it and we can get it back anytime :)

**Comment by chaudum** [comment]
ack. please squash to a single commit!
